modules/spo-log-enricher-app-trace.adoc

```diff
@@ -32,13 +32,18 @@ kind: Pod
 metadata:
   name: log-pod
 spec:
+  # TODO: How to update this one that alread has a seccompProfile defined? Is the securityContext on the container correct?
```

Comment:
@s-urbaniak Another one that already has securityContext, what to do here?

Reply:
same as above

Reply:
@stlaz Same question as above - I just added in `runAsNonRoot: true` to the top, was that correct?